Feature/jn s4eb jb 5 form page

an-nguyen-jh opened the pull request.

an-nguyen-jh committed:
[Style] Create layout for employee page and rename file

Create layout of emloyee page include: appbar and content

Rename nomalize.css to normalize.css

Change primaryColor of page

an-nguyen-jh committed:
[Style] Show name of form in Appbar

an-nguyen-jh committed:
[Fix] Move singout button to appbar

an-nguyen-jh committed:
[Style] Create frame and all input field in device info form

an-nguyen-jh committed:
[Feat] Create iamgeUpload btn and list of preview image

an-nguyen-jh committed:
[Feat] Load inputted value from firebase

an-nguyen-jh committed:
[Feat] Preview images and upload images to firebase

Preview image before upload

Upload image to firebase and store images URL in form info

an-nguyen-jh committed:
[Feat] Change logic of upload image and add notification

Upload image when have new chosen images and delete old images

Notice when using complete form

an-nguyen-jh committed:
[Feat] Clear form info (not include image)

an-nguyen-jh committed:
[Style] Frame for request device form

Create frame for request device form

change style sheet in form.css

an-nguyen-jh committed:
[Feat] Create new device request and store to firebase

an-nguyen-jh committed:
[Style] Create Select and UploadImage components

an-nguyen-jh committed:
[Fix] Remove console.log and add default for prevewImages

an-nguyen-jh committed:
[Refactor] Remove unnecessary things and rename variable

Remove unnecessary files, lines, comment

Rename wrong and unclear variable name

Use Suspense wrap lazy import component

an-nguyen-jh committed:
[Fix] Remove unneccessary Suspense

anh-dinh-jh commented on src/components/Appbar/Appbar.js:
```diff
@@ -0,0 +1,117 @@
+  }
+
+  componentDidUpdate(prevProps) {
+    const subPathname = this.props.location.pathname.split("/")[2];
+    if (subPathname !== prevProps.location.pathname.split("/")[2]) {
```
Should have a separate function for string splitting

anh-dinh-jh commented on src/utils/routerHandler.js:
```diff
@@ -0,0 +1,3 @@
+export function getRouterTile(routers, path) {
```
Incorrect name

anh-dinh-jh commented on src/utils/parser.js:
```diff
@@ -0,0 +1,7 @@
+import DEV_CONFIG from "../config";
+const URLFirebase = DEV_CONFIG.URL_FIRESTORE;
+export default function parsefullFileURL(fileURL) {
```
Incorrect naming

anh-dinh-jh commented on .vscode/settings.json:
```diff
@@ -0,0 +1 @@
+{}
```
Redundant file

an-nguyen-jh committed:
[Fix] Fix mistakes in code review

Remove redundant file

Rename incorrect variable names

Create function for split url

an-nguyen-jh committed:
[Feat] Create custom class for DeviceInfo and Device DeviceRequest

Use custom class to create default property

Create converter but no effect until now

an-nguyen-jh committed:
[Fix] Fix deviate input field when have error
